cgi-bin/LJ/User/Account.pm

```diff
@@ -994,8 +994,26 @@ sub load_existing_identity_user {
     my ($type, $ident) = @_;
 
     my $dbh = LJ::get_db_reader();
-    my $uid = $dbh->selectrow_array("SELECT userid FROM identitymap WHERE idtype=? AND identity=?",
-                                    undef, $type, $ident);
+    my $uid;
+
+    # if given an https URL, also look for existing http account
+    # (we should have stripped the protocol before storing these, sigh)
```

Comment:
There would be at least one collusion, from the accounts that the support request was referring to.